AirCasting/Utils/NetworkChecker.swift

```diff
@@ -0,0 +1,37 @@
+// Created by Lunar on 08/07/2021.
+//
+
+import Foundation
+import Network
+
+protocol CheckNetwork {
+    func monitorNetwork()
```

Comment:
`connectionAvailable` should be inside the protocol since it's the most crucial part of network checking I think :)